### ISSUE TITLE
Reintegration of tobac-tutorials into main repository

### ISSUE THREAD

**Issue by lettlini** [tobac-project/tobac#282]
I think we have come to the consensus to reintegrate the tobac-tutorials into the main repository. I would like to use this issue to track progress and discuss details if necessary.

There is already a big overlap between `tobac/examples` and `tobac-tutorials`:

**exists in both repositories**:
- Example_OLR_Tracking_model
- Example_OLR_Tracking_satellite
- Example_Precip_Tracking
- Example_Updraft_Tracking

**currently only exists in tobac/tobac-tutorials**:
- Basics
- Example_VIS_Tracking_Satellite
- Example_Updraft_Tracking

I think it would be a good idea to start with the examples provided in `Basics/` as these notebooks are a great resource for getting started with tobac.

**Comment by snilsn** [comment]
I think the first step should be to process the PRs in the tutorial repository. I will start by reviewing [PR 27](https://github.com/tobac-project/tobac-tutorials/pull/27)

**Comment by lettlini** [comment]
We now have processed all pending PRs in the `tobac-tutorials` repository (thanks for your efforts @snilsn!). 

I think it is time we start thinking about how and when we want to integrate these notebooks into the main repository.

**Comment by fsenf** [comment]
I will now start to work on the re-integration of the tutorial notebooks

This is the current content
```
> git ls-files 
.github/pull_request_template.md
.readthedocs.yml
LICENSE
Makefile
README.md
conf.py
docs/RunningTutorials.md
docs/Testing-Sphinx-based-Rendering.md
index.rst
requirements.txt
tutorials/Basics/Idealized-Case-1_Tracking-of-a-Test-Blob-in-2D.ipynb
tutorials/Basics/Idealized-Case-2_Two_crossing_Blobs.ipynb
tutorials/Basics/Methods-and-Parameters-for-Feature-Detection_Part_1.ipynb
tutorials/Basics/Methods-and-Parameters-for-Feature-Detection_Part_2.ipynb
tutorials/Basics/Methods-and-Parameters-for-Linking.ipynb
tutorials/Basics/Methods-and-Parameters-for-Segmentation.ipynb
tutorials/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb
tutorials/Example_OLR_Tracking_satellite/Example_OLR_Tracking_satellite.ipynb
tutorials/Example_Precip_Tracking/Example_Precip_Tracking.ipynb
tutorials/Example_Updraft_Tracking/Example_Updraft_Tracking.ipynb
tutorials/Example_VIS_Tracking_Satellite/Example_VIS_Tracking_Satellite.ipynb
tutorials/Example_vorticity_tracking_model/Example_vorticity_tracking_model.ipynb
```

Two things to care for:
1. integrate notebooks
2. don't loose other valuable content like requirements, PR-Template and docs - etc.

**Comment by w-k-jones** [comment]
Resolved with #334
